cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/etl/common/TransformDetail.java

```diff
@@ -16,40 +16,40 @@
 
 package co.cask.cdap.etl.common;
 
-import co.cask.cdap.etl.api.StageMetrics;
+import co.cask.cdap.api.metrics.Metrics;
 import co.cask.cdap.etl.api.Transformation;
 
+import java.util.Map;
+
 /**
  * Class that encapsulates {@link co.cask.cdap.etl.api.Transform} and transformId
```

Comment:
looks like this class is changing to represent all transforms in a pipeline rather than a single transform. Lets update the javadocs and class name so that this is more clear.